#154953778 How to present deactivated users in gym overview

hadijahkyampeire opened the pull request:
#### What does this PR do?
Decide how to present deactivated users in gym overview

#### Description of Task to be completed?
List only active members in the default list and have a second overview page for the deactivated ones.

#### How should this be manually tested?
When you access the gym users view, you can use the toggle button to see the inactive and active gym users.

#### Any background context you want to provide?
For one to know whether a gym user is active or inactive, they would click on the user to view their status.

#### What are the relevant pivotal tracker stories?

[154953778](https://www.pivotaltracker.com/story/show/154953778)
#### Screenshots (if appropriate)
<img width="1438" alt="screen shot 2018-02-14 at 15 27 10" src="https://user-images.githubusercontent.com/30952856/36204257-a36b1ea0-119b-11e8-9927-80ef4d72aa23.png">
<img width="1440" alt="screen shot 2018-02-14 at 15 27 23" src="https://user-images.githubusercontent.com/30952856/36204271-aef632fa-119b-11e8-8e0e-67057647edf0.png">

hadijahkyampeire commented:
@Mercy-Muchai My branch is ready for merging I have solved all the rebase conflicts

hadijahkyampeire commented:
@Mercy-Muchai I have also made the final squash, you can see it below the comment

muchai-mercy commented:
Still two commits. Please squash

muchai-mercy commented on .gitignore:
```diff
@@ -65,3 +65,4 @@ node_modules
 venv-django
```
Do we need to have this and line 65 too?

hadijahkyampeire committed:
[feature 154953778] present-deactivated-gym-users:
-Modify requirements_devel.txt
-Modify settings.py
-Add new template for inactive gym users
-Add toggle buttons for filtering different users
-Add tests for deactivated gym users
-Add url for deactivated gym users
-Add class view for deactivated gym users

[Delivers #154953778]

muchai-mercy reviewed (approved):
LGTM
